run

```diff
@@ -74,6 +82,15 @@ def parse_args():
     parser.add_argument('--swiftc',
                         metavar='PATH',
                         help='swiftc executable')
+    clang_arguments = parser.add_mutually_exclusive_group()
+    clang_arguments.add_argument('--clang-source-path',
+                                 metavar="PATH",
+                                 help='path to llvm-project to build',
+                                 type=os.path.abspath)
+    clang_arguments.add_argument('--clangc',
+                                 metavar="PATH",
+                                 help='clang executable',
```

Comment:
Suggestion for improved message:
```suggestion
                                 help='If specified uses the given clang executable to build the suite',
```

Reply:
Just a heads-up that I suggested an improved message here. If you prefer your one, just comment.

Reply:
I updated it to be "clang executable to build Xcode projects".